### ISSUE TITLE
Vite build fails with ReferenceError: module is not defined. Node 23

### ISSUE THREAD

**Issue by julianrubisch** [maglevhq/maglev-core#96]
I think I'm seeing somewhat of a regression of https://github.com/maglevhq/maglev-core/issues/33 IIUC.

Going through the Getting Started guide and then browsing to http://localhost:3000/maglev/editor/en/index gives me a 

```
file:///.../maglev-core-cbb74a1b63f9/tailwind.config.js:9
  module.exports = {
  ^

  ReferenceError: module is not defined
      at file:///.../maglev-core-cbb74a1b63f9/tailwind.config.js:9:1
   ...

  Node.js v23.3.0
```

The same happens when running `bin/vite build --clear --mode=development` as suggested by the error message.

**Further Context:** It seems to be the node version. Running it with Node v20 works! 🎉

**Comment by did** [comment]
thanks @julianrubisch for reporting this. More information here: https://github.com/tailwindlabs/tailwindcss/issues/14818.

I'm starting to wonder that migrating to Hotwire / Stimulus won't be a bad move after all...

**Comment by julianrubisch** [comment]
Ah yeah that makes sense. FWIW I forked it and replaced module.exports by export default which got it working again…

**Comment by did** [comment]
fixed by the PR #99.
